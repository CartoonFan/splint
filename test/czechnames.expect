
Finished checking --- no warnings

czechnames.c:5:5: Variable michelob_light name violates Czech naming
    convention.  Czech prefix michelob is not the name of a type.
  Variable name is not consistent with Czech naming convention. (Setting either
  -czechvars or -namechecks will suppress message)
czechnames.c:26:5: Function budweiser name is not consistent with Czech naming
                      convention.  Accessible types: pivo
  Function or iterator name is not consistent with Czech naming convention.
  (Setting either -czechfcns or -namechecks will suppress message)

Finished checking --- 2 code warnings, as expected

czechnames.c:26:5: Function budweiser name is not consistent with Czech naming
                      convention.  Accessible types: pivo
  Function or iterator name is not consistent with Czech naming convention.
  (Setting either -czechfcns or -namechecks will suppress message)

Finished checking --- 1 code warning, as expected

czechnames.c:4:5: Variable pivo_pyet name violates Czech naming convention.
    Czech prefix pivo names an abstract type that is not accessible.
  Use +accessczech to allow access to type <t> in functions named <t>_<name>.
czechnames.c:5:5: Variable michelob_light name violates Czech naming
    convention.  Czech prefix michelob is not the name of a type.
  Variable name is not consistent with Czech naming convention. (Setting either
  -czechvars or -namechecks will suppress message)
czechnames.c:8:17: Constant pivo_devenast name violates Czech naming
    convention. Czech prefix pivo names an abstract type that is not
    accessible.
  Use +accessczech to allow access to type <t> in functions named <t>_<name>.
czechnames.c:11:6: Function pivo_yedno name violates Czech naming convention.
    Czech prefix pivo names an abstract type that is not accessible.
  Use +accessczech to allow access to type <t> in functions named <t>_<name>.
czechnames.c: (in function pivo_yedno)
czechnames.c:15:11: Left operand of == is abstract type (pivo): p == 1
  An abstraction barrier is broken. If necessary, use /*@access <type>@*/ to
  allow access to an abstract type. (-abstract will suppress message)
czechnames.c:26:5: Function budweiser name is not consistent with Czech naming
                      convention.  Accessible types: pivo

Finished checking --- 6 code warnings, as expected
