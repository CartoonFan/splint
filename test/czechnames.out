gmake[3]: Entering directory `/tmp/LCLintDev/test'

Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.

Finished LCLint checking --- no code errors found

Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
czechnames.c:5:5: Variable michelob_light name violates Czech naming
    convention.  Czech prefix michelob is not the name of a type.
  Variable name is not consistent with Czech naming convention. (Setting either
  -czechvars or -namechecks will suppress message)
czechnames.c:26:5: Function budweiser name is not consistent with Czech naming
                      convention.  Accessible types: pivo
  Function or iterator name is not consistent with Czech naming convention.
  (Setting either -czechfcns or -namechecks will suppress message)

Finished LCLint checking --- 2 code errors found, as expected

Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
czechnames.c:26:5: Function budweiser name is not consistent with Czech naming
                      convention.  Accessible types: pivo
  Function or iterator name is not consistent with Czech naming convention.
  (Setting either -czechfcns or -namechecks will suppress message)

Finished LCLint checking --- 1 code error found, as expected

Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
czechnames.c:4:5: Variable pivo_pyet name violates Czech naming convention.
    Czech prefix pivo names an abstract type that is not accessible.
  Use +accessczech to allow access to type <t> in functions named <t>_<name>.
czechnames.c:5:5: Variable michelob_light name violates Czech naming
    convention.  Czech prefix michelob is not the name of a type.
  Variable name is not consistent with Czech naming convention. (Setting either
  -czechvars or -namechecks will suppress message)
czechnames.c:8:17: Constant pivo_devenast name violates Czech naming
    convention. Czech prefix pivo names an abstract type that is not
    accessible.
  Use +accessczech to allow access to type <t> in functions named <t>_<name>.
czechnames.c:11:6: Function pivo_yedno name violates Czech naming convention.
    Czech prefix pivo names an abstract type that is not accessible.
  Use +accessczech to allow access to type <t> in functions named <t>_<name>.
czechnames.c: (in function pivo_yedno)
czechnames.c:15:11: Left operand of == is abstract type (pivo): p == 1
  An abstraction barrier is broken. If necessary, use /*@access <type>@*/ to
  allow access to an abstract type. (-abstract will suppress message)
czechnames.c:26:5: Function budweiser name is not consistent with Czech naming
                      convention.  Accessible types: pivo

Finished LCLint checking --- 6 code errors found, as expected
gmake[3]: Leaving directory `/tmp/LCLintDev/test'
